CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+## Set up
+
+After cloning this repo, create a [virtualenv](https://virtualenv.pypa.io/en/stable/) and ensure dependencies are installed by running:
+
+```sh
+virtualenv venv
+source venv/bin/activate
+make dev-setup
+git checkout -B YOUR_BRANCH_NAME
+```
+
+You are not ready to start developing
```

Comment:
Why did I do this then? 😛

Reply:
Hah I just copy/pasted the original docs